### PR TITLE
Alexandra bug

### PR DESCRIPTION
**Pull request by malexa21**
As requested I fixed the list of bugs shown in the readme of the app, as well as another bug that I found reffering to the duration of a singular activity. Please provide me with feedback for the code that I written.

### CODE REVIEW

File: src/views/WaterTrackingView.jsx [hunk 1]
```diff
@@ -53,18 +53,18 @@ export const WaterTrackingView = () => {
...
               style={{ color: getProgressColor(currentIntake, WATER_GOAL) }}
             >
-              {WATER_GOAL} / {currentIntake} / {WATER_GOAL}
+              {currentIntake} / {WATER_GOAL}
             </span>{" "}
             glasses of water
           </Title>
           <Flex gap="small">
-            <Button type="primary" onClick={() => handleWaterIntakeUpdate(-1)}>
+            <Button type="primary" onClick={() => handleWaterIntakeUpdate(+1)}>
```
Comment (cezarmocanu) [inline src/views/WaterTrackingView.jsx]: plus sign is not required here, only 1 is enough

File: src/views/MenuView.jsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const MenuView = () => {
 
   const handleMenuClick = (event) => {
     const targetPage = event.key;
-    navigate(`/targetPage`);
+    navigate(`/` + targetPage);
```
Comment (cezarmocanu) [inline src/views/MenuView.jsx]: you can use string template as follows

```

`${targetPage}\`

```

File: src/views/ActivityTrackingView.jsx [hunk 3]
```diff
@@ -40,50 +40,49 @@ export const ActivityTrackingView = () => {
...
     if (duration <= 0) {
       messageInstance.error(`Duration needs to be at least 1 minute`);
       return;
     }
 
     messageInstance.success(`Added ${selectedActivity} - ${duration} min`);
-    setDuration(Math.random() * 999999);
-    setSelectedActivity("");
+    setDuration(0);
+    setSelectedActivity(null);
```
Comment (cezarmocanu) [inline src/views/ActivityTrackingView.jsx]: here "" should remain as the initial value is ""

**Review by cezarmocanu — CHANGES_REQUESTED** [review]

File: src/views/MenuView.jsx (hunk 2 above)
Comment (malexa21) [reply src/views/MenuView.jsx]: cred ca ai pus / gresit acolo, am incercat cu \ dar nu a functionat

**Review by cezarmocanu — APPROVED** [review]

**Pull request merged**